Mod Menu Helper/assets/modmenu/lang/ko_kr.json

```diff
@@ -0,0 +1,55 @@
+    "modmenu.summaryTranslation.cem": "Optifine의 커스텀 엔티티 (알파)\n§9✎§8 옵티파인 지원, 모델 고침",
+    "modmenu.summaryTranslation.dcch": "여러 월드에 걸쳐 입력된 기록 유지\n§8✎ (옵션 없음)",
+    "modmenu.summaryTranslation.dynamicfps": "백그라운드에서 FPS 감소\n§9✎§8 모드 토글, 호버시 복구, ...",
+    "modmenu.summaryTranslation.enhancedblockentities": "빠른 상자, 침대, 표지판 등 렌더\n§9✎§8 개선할 블록, 메소드",
+    "modmenu.summaryTranslation.entityculling": "벽 뒤 엔티티 렌더 안함\n§8✎ (고급 옵션만 있음)",
+    "modmenu.summaryTranslation.entity_texture_features": "Optifine의 랜덤/Emissive 몹\n§9✎§8 랜덤 몹, Emissivity, 눈 깜빡임, ...",
+    "modmenu.summaryTranslation.fabrishot": "F9을 눌러 HD 스크린샷 생성\n§9✎§8 F2 덮어쓰기, 화질, 키바인드, ...",
+    "modmenu.summaryTranslation.farsight": "§c*§r 서버에서 계속 청크가 보이도록 로드\n§8✎ (옵션 없음)",
+    "modmenu.summaryTranslation.fastchest": "§c*§r 상자 렌더 제거\n§9✎§8 모드 토글",
+    "modmenu.summaryTranslation.fastopenlinksandfolders": "링크/폴더를 열 때 응답 없음 제거\n§8✎ (옵션 없음)",
+    "modmenu.summaryTranslation.ferritecore": "RAM 사용량 감소\n§8✎ (고급 옵션만 있음)",
+    "modmenu.summaryTranslation.forgetmechunk": "청크 보더에서 랙 스파이크 제거\n§8✎ (옵션 없음)",
+    "modmenu.summaryTranslation.hydrogen": "RAM 사용량 감소\n§8✎ (옵션 없음)",
+    "modmenu.summaryTranslation.iris": "Optifine의 셰이더\n§9✎§8 모드 토글, 셰이더 선택, 키바인드, ...",
+    "modmenu.summaryTranslation.itemmodelfix": "들고있는 아이템의 투명한 선 제거\n§9✎§8 메소드",
+    "modmenu.summaryTranslation.lambdabettergrass": "Optifine의 \"개선된 잔디/눈\"\n§9✎§8 퀄리티, 눈, 리소스 팩",
+    "modmenu.summaryTranslation.lambdynlights": "Optifine의 \"동적 조명\"\n§9✎§8 퀄리티, 엔티티 옵션, ...",
+    "modmenu.summaryTranslation.languagereload": "빠른 언어 교체, 검색 바\n§9✎§8 F3+J 로 언어 다시 로드",
+    "modmenu.summaryTranslation.lazydfu": "빠른 게임 시작\n§8✎ (옵션 없음)",
+    "modmenu.summaryTranslation.lithium": "물리와 청크 최적화\n§8✎ (고급 옵션만 있음)",
+    "modmenu.summaryTranslation.modmenu": "이 모드 메뉴\n§9✎§8 버튼 배치, 필터링, 정렬, ...",
+    "modmenu.summaryTranslation.morechathistory": "최대 채팅 히스토리 증가\n§8✎ (옵션 없음)",
+    "modmenu.summaryTranslation.nofade": "로딩 화면 Fade 애니메이션 제거\n§8✎ (옵션 없음)",
+    "modmenu.summaryTranslation.notenoughcrashes": "충돌 후에 리포트 생성/계속 플레이\n§8✎ (고급 옵션만 있음)",
+    "modmenu.summaryTranslation.phosphor": "§c*§r 조명, 청크 개선\n§8✎ (고급 옵션만 있음)",
+    "modmenu.summaryTranslation.puzzle": "일부 Optifine 기능\n§9✎§8 구성하려면 모드를 확대하세요 ( ▼ 아이콘 클릭)",
+    "modmenu.summaryTranslation.puzzle-base": "일부 Optifine 기능\n§9✎§8 로딩 화면 스플래시, ...",
+    "modmenu.summaryTranslation.smoothboot": "게임 응답 없음 감소\n§8✎ (고급 옵션만 있음)",
+    "modmenu.summaryTranslation.sodium": "렌더링, FPS 최적화\n§9✎§8 비디오 설정에서: 최적화, ...",
+    "modmenu.summaryTranslation.starlight": "조명, 청크 개선\n§8✎ (옵션 없음)",
+    "modmenu.summaryTranslation.reeses-sodium-options": "수직 비디오 설정\n§8✎ (옵션 없음)",
+    "modmenu.summaryTranslation.sodium-extra": "Optifine의 그래픽 설정\n§9✎§8 비디오 설정에서: 애니메이션, 파티클, ...",
+    "modmenu.summaryTranslation.tiefix": "Telementary 비활성화, 일부 오류 해결\n§9✎§8 해결할 오류, Telemantray 토글",
+    "modmenu.summaryTranslation.tooltipfix": "Tooltip overflow 차단\n§8✎ (옵션 없음)",
```

Comment:
See images on https://www.curseforge.com/minecraft/mc-mods/tooltipfix for context